public/src/string-class.js

```diff
@@ -0,0 +1,137 @@
+' use strict';
+
+const StringClass = {
+  /**
```

Comment:
### Nit-Pick
Kes want us to follow the below comment's format 

/**
 * Name of function
 *
 * Short description
 *
 * @return{...}
 */